app/src/main/java/org/wikipedia/analytics/eventplatform/ImageRecommendationsEvent.kt

```diff
@@ -0,0 +1,60 @@
+package org.wikipedia.analytics.eventplatform
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import org.wikipedia.WikipediaApp
+import org.wikipedia.descriptions.DescriptionEditActivity
+import org.wikipedia.util.ActiveTimer
+import java.net.URLEncoder
+
+@Serializable
+@SerialName("/analytics/mobile_apps/android_image_recommendation_event/1.0.0")
+class ImageRecommendationsEvent(
+   private val action: String,
+   private val active_interface: String,
+   private val action_data: String,
+   private val primary_language: String,
+   private val wiki_id: String
+) : MobileAppsEvent(STREAM_NAME) {
```

Comment:
Not sure if the parameters are unused or not. If not, please add `@Suppress("unused")`